### PR TITLE
index page view done but not linked to elements yet.

### PR DESCRIPTION
**Pull request by MorganGThorpe**

### CODE REVIEW

File: app/views/tools/index.html.erb [hunk 1]
```diff
@@ -1,12 +1,75 @@
...
   <div class= "row m-0">
     <div class="col-xs-12 col-sm-6 m-0 w-100">
       <h2>tool researched</h2>
-      <p>col6</p>
+      <h5>paragraph 2</h5>
+      <div class= "form-row align-items-center my-4">
+        <div class= "form-group col-4 m-1">
+          <p>Category</p>
+          <select class="form-control ">
+            <option>Gardening</option>
```
Comment (helenagagern) [inline app/views/tools/index.html.erb:10]: We are wondering if we can write this in ruby. Either with an each iteration over categories or with somethin g like this <%= f.input :category, collection: Restaurant::CATEGORIES %>

Comment (MorganGThorpe) [reply app/views/tools/index.html.erb:10]: Yes! I was just making a model that we can replace with Ruby when we have seeded the db and have some tools to show.

File: app/views/tools/index.html.erb [hunk 2]
```diff
@@ -1,12 +1,85 @@
-<p>This is totally working</p>
 <div class= "container m-0 mw-100">
```
Comment (lucien-george) [inline app/views/tools/index.html.erb:1]: remove container class

File: app/views/tools/index.html.erb [hunk 3]
```diff
@@ -1,12 +1,85 @@
...
   <div class= "row m-0">
     <div class="col-xs-12 col-sm-6 m-0 w-100">
```
Comment (lucien-george) [inline app/views/tools/index.html.erb:3]: kill the padding

File: app/views/tools/index.html.erb [hunk 4]
```diff
@@ -1,12 +1,85 @@
...
+              <p>Tool owner description</p>
+            </div>
+          </div>
+        </div>
+      </div>
+
     </div>
-    <div class="d-none d-sm-block col-sm-6 m-0 w-100">
-      <p>map</p>
+    <div class="d-none d-sm-block col-sm-6 m-0 w-100" >
```
Comment (lucien-george) [inline app/views/tools/index.html.erb:81]: kill the padding

File: app/views/tools/index.html.erb (hunk 2 above)
Comment (MorganGThorpe) [reply app/views/tools/index.html.erb]: Done in latest push

File: app/views/tools/index.html.erb (hunk 3 above)
Comment (MorganGThorpe) [reply app/views/tools/index.html.erb]: Done in lates push

File: app/views/tools/index.html.erb (hunk 4 above)
Comment (MorganGThorpe) [reply app/views/tools/index.html.erb]: Done in latest push

**Pull request merged**